mleap-databricks-runtime-fat-tests/genTestCode.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+rm -rf mleap-databricks-runtime-fat-tests/src
+mkdir -p mleap-databricks-runtime-fat-tests/src/test
+
+cp -R mleap-core/src/test mleap-databricks-runtime-fat-tests/src
```

Comment:
Does it work if you specify test source locations using sbt?

Reply:
It does not work. Sbt cannot run test code inside dependency jars.
Another work around is add those test class jars as dependency and add test classes which extends original test classes, but this approach is more complex.

Reply:
Oh, do you mean specify test source dir to be dir inside in other sub-project ? Let me try...